gapipy/cache.py

```diff
@@ -168,3 +175,26 @@ def info(self):
 
     def is_cached(self, key):
         return self._client.exists(self.key_prefix + key)
+
+
+if django_settings:
+    class DjangoCache(BaseCache):
+        def __init__(self, *args, **kwargs):
+            assert django_settings.CACHES is not None, "No CACHES found in django settings!"
+            assert django_settings.CACHES.get('gapi') is not None, "No 'gapi' entry found in settings.CACHES!"
+            self.client = django_cache['gapi']
+
+        def clear(self):
+            self.client.clear()
+
+        def delete(self, key):
+            del self.client[key]
```

Comment:
I think this is `client.delete(key)` in Django terms 🤔

Reply:
There was no values to it in the olden days, but since 3.1 yes, there is a boolean return so I'll swap